src/main.rs

```diff
@@ -654,11 +664,17 @@ impl Fastmod {
         dirs: Vec<&str>,
         file_set: Option<FileSet>,
         hidden: bool,
+        no_ignore: bool,
         changed_files: Option<Vec<PathBuf>>,
         visited: Option<HashSet<PathBuf>>,
     ) -> Result<()> {
         let walk = walk_builder_with_file_set(dirs, file_set)?
             .hidden(!hidden)
+            .parents(!no_ignore)
```

Comment:
I'm surprised that we need all these, but based on rg --help it looks like we do.

Reply:
There is a simple single method to replace all these https://docs.rs/ignore/latest/ignore/struct.WalkBuilder.html#method.standard_filters.

this could probably be just:

```rust
            .standard_filters(!no_ignore)
            .hidden(!hidden)
```

I was too lazy to think about how the option should behave with the --hidden flag (